cookbooks/travis_ci_ruby/attributes/default.rb

```diff
@@ -1,11 +1,11 @@
-override['rvm']['default'] = '2.2.5'
+override['rvm']['default'] = '2.2.6'
 override['rvm']['rubies'] = %w(
   1.9.2
   1.9.3
   2.0.0
   2.1.10
-  2.2.5
-  2.3.1
```

Comment:
@meatballhat Yeah, we're in the process of moving some of our projects to 2.3.3, but others are still on 2.3.1. I definitely wouldn't expect those projects to take the ~30s hit the same week the new version comes out. Maybe a month of transition time at a minimum seems good? 

Just out of curiosity, would it make the images too large to simply pre-install all ruby versions so no one suffers the dynamic installation penalty?